pkg/platform/runtime/setup/implementations/installer/runtime.go

```diff
@@ -0,0 +1,32 @@
+package installer
```

Comment:
I thought we discussed and agreed that we do not want a separate setup implementation to facilitate offline installs. It should use the exact same mechanic as online builds, and we want a new layer that communicates the information that can be either offline or online.

This will require refactoring of our setup implementations to split out any sort of resourcing of bits to this layer. But this should not translate into us having online and offline setup implementations.

Reply:
As discussed on Zoom, we'll leave the larger refactor for later. For now, the offline installer capability is more important.